Sources/MapboxDirections/AdministrativeRegion.swift

```diff
@@ -1,7 +1,6 @@
 import Foundation
 
 /**
- :nodoc:
  `AdministrativeRegion` describes corresponding object on the route.
```

Comment:
If we publicly document this type, then the documentation should probably mention the `Intersection.regionCode` property that is of the simpler String type and that this type is primarily used for encoding and decoding purposes.

Reply:
Added mention to the doc